pkg/hive/cell/config.go

```diff
@@ -165,3 +166,44 @@ func stringToCIDRHookFunc(from reflect.Type, to reflect.Type, data interface{})
 	}
 	return cidr.ParseCIDR(s)
 }
+
+// stringToSliceHookFunc converts strings to []string by splitting by spaces or commas.
+func stringToSliceHookFunc(from reflect.Type, to reflect.Type, data interface{}) (interface{}, error) {
+	if from.Kind() != reflect.String || to.Kind() != reflect.Slice {
+		return data, nil
+	}
+
+	raw := data.(string)
+	if raw == "" {
+		return []string{}, nil
+	}
+
+	// Split by commas first
+	if ss := strings.Split(raw, ","); len(ss) > 1 {
```

Comment:
What about cases where its a slice of size 1?

Reply:
I guess we just fallthrough to the whitespace case,  I suppose quoted strings with whitespaces aren't supported?

Reply:
Hey, what do you mean by quoted strings? If len > 1, the string contains at least one comma, so comma splitting is chosen, and execution ends here. Without commas, a space-delimited string is assumed.

Reply:
Quoted strings aren't supported. You can either split by commas or by spaces, but not by both, with commas taking precedence: `foo,bar baz` => `[]string{"foo", "bar baz"}`.

Reply:
One thing I'm still confused about, if this: `foo,bar baz,zap` => `[]string{"foo", "bar baz", "zap"}` is a valid input.   Then I would expect any subset of this to also be valid, i.e.

`bar baz` should be valid and produce => `[]string{"bar baz"}` but in this case this would produce: `[]string{"bar", "baz"}`.

Reply:
The point is to prefer splitting by commas and just have the whitespace splitting as fallback for compatibility reasons. Eventually we want to get rid of splitting by whitespace.